client/src/pages/ContactMe/ContactMe.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useRef } from 'react';
+import emailjs from '@emailjs/browser';
+import { useNavigate } from 'react-router-dom';
+
+export const ContactMe = () => {
+  const navigate = useNavigate();
+  const form = useRef();
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    emailjs
+      .sendForm(
+        'service_3ceepzv',
+        'template_n1euugs',
+        form.current,
+        '1d-j2JW1QjE-26RcM',
+      )
+      .then(
+        (result) => {
+          console.log(result.text);
+        },
+        (error) => {
+          console.log(error.text);
+        },
+      );
+    e.target.reset();
+    navigate('/');
+  };
+
+  return (
+    <div className="bg-white rounded-md shadow-md py-6 px-8 mx-auto max-w-xl">
+      <h2 className="text-2xl font-medium mb-4 text-purple-600">Contact Me</h2>
+      <form ref={form} onSubmit={sendEmail}>
+        <div className="mb-4">
+          <label
+            htmlFor="name"
+            className="block text-gray-700 font-medium mb-2"
+          >
+            {/* Name */}
+          </label>
+          <input
+            id="fname"
+            name="firstname"
+            type="text"
+            className="w-full border border-gray-400 py-2 px-3 rounded-md text-gray-700 leading-tight focus:outline-none focus:border-purple-600"
+            placeholder="Your name"
+            required
+          />
+        </div>
+        <div className="mb-4">
+          <label
+            htmlFor="email"
+            className="block text-gray-700 font-medium mb-2"
+          >
+            {/* Email */}
+          </label>
+          <input
+            id="mailid"
```

Comment:
id and htmlFor should have the same value

Reply:
i'll change it now